src/libs/ReportUtils.js

```diff
@@ -917,36 +917,34 @@ function buildOptimisticChatReport(
  */
 function buildOptimisticCreatedReportAction(ownerEmail) {
     return {
-        0: {
```

Comment:
> I see we enclose the returned object in another object keyed by the ID every time we call this method, should we change this so that this method does it?

I much prefer the way it is now because `buildOptimisticCreatedReportAction` just returns the reportAction as expected, not an object like this:

```
reportActionID: { ...reportAction }
```

The previous design was quite unintuitive imo, and I saw people using it incorrectly in  PR reviews before I adjusted this:

```js
// Before this change, this was WRONG, and that's confusing imo
const createdReportAction = ReportUtils.buildOptimisticCreatedReportAction();
```

Reply:
Going to plan on leaving this as-is unless you feel very strongly that I should change it.

Reply:
ok